docs/wrapperkernels.rst

```diff
@@ -68,6 +68,8 @@ To launch your kernel, add this at the end of your module::
         from ipyernel.kernelapp import IPKernelApp
         IPKernelApp.launch_instance(kernel_class=MyKernel)
 
+Place your kernel anywhere that you could import it (e.g. current directory). Now create a JSON kernel spec file (see below for example), and place it in ``~/.ipython/kernels/<mykernelname>/``. Now, you can run your kernel using ``ipython console --kernel <mykernelname>``
```

Comment:
The location to put it in now [depends on the system](http://jupyter.readthedocs.org/en/latest/system.html#data-files). People could use the `jupyter kernelspec install` command to put it in the right place.

Also, `ipython console` -> `jupyter console`